mautic/mautic
# 00-Rector: Upgrade code to PHP8.0

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

I managed to run rector config set to upgrade the code base to be `php8.0` compatible, this PR is the first in a series of PRs that upgrade each bundle on its own. 

This PR fixes all the issues I encountered in #12322, once this is merged I will start running the rules for all the other bundles in sequence  



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Code review is required only
